src/style.css

```
@@ -1 +1,84 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  background-color: #e5e5e5;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  height: 60px;
+}
+
+.logo__img {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  display: block;
+}
+
+.nav {
+  display: inline-block;
+  height: 100%;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  height: 100%;
+}
+
+.nav__link {
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  color: #000;
+  height: 100%;
+  display: flex;
+  justify-items: center;
+  align-items: center;
+}
+
+.nav__item {
+  margin-right: 20px;
+  height: 100%;
+}
+
+.nav__link:last-of-type {
+  margin-right: 0;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00acdc;
+  display: block;
+  position: absolute;
+  bottom: 0;
+  width: 37px;
```

should be the full size
```suggestion
  width: 100%;
```